packages/lib/buildCalEventFromBooking.ts

```diff
@@ -0,0 +1,106 @@
+import type { Prisma } from "@prisma/client";
+
+import dayjs from "@calcom/dayjs";
+
+import { parseRecurringEvent } from "./isRecurringEvent";
+import { getTranslation } from "./server";
+
+type DestinationCalendar = {
+  id: number;
+  integration: string;
+  externalId: string;
+  primaryEmail: string | null;
+  userId: number | null;
+  eventTypeId: number | null;
+  credentialId: number | null;
+} | null;
+
+type Attendee = {
+  email: string;
+  name: string;
+  timeZone: string;
+  locale: string | null;
+};
+
+type Organizer = {
+  email: string;
+  name: string | null;
+  timeZone: string;
+  locale: string | null;
+};
+
+type EventType = {
+  title: string;
+  recurringEvent: Prisma.JsonValue | null;
+  seatsPerTimeSlot: number | null;
+  seatsShowAttendees: boolean | null;
+};
+
+type Booking = {
+  title: string;
+  description: string | null;
+  startTime: Date | null;
+  endTime: Date | null;
+  userPrimaryEmail: string | null;
+  uid: string;
+  destinationCalendar: DestinationCalendar;
+  user: {
+    destinationCalendar: DestinationCalendar;
+  } | null;
+  attendees: Attendee[];
+  eventType: EventType | null;
+};
+
+export const buildCalEventFromBooking = async ({
+  booking,
+  organizer,
+  location,
+  conferenceCredentialId,
+}: {
```

Comment:
I'm seeing that the result of this is being passed into the `EventManager`. Should we add team members to the result object?

Reply:
If team members are passed at other places, then we should pass those here too.